tests/LevelTest.php

```diff
@@ -7,17 +7,22 @@ class LevelTest extends \PHPUnit_Framework_TestCase
 {
     public function testLevel()
     {
-        $l = Level::TEST();
-        $this->assertNull($l);
+        try {
+            $level = Level::TEST();
+            $this->fail();
+        } catch (\Exception $exception) {
+            $this->assertTrue(true);
```

Comment:
This appears to be testing that `Level::TEST()` throws an exception. The alternative would be to use `expectException()`